packages/@dcl/inspector/src/lib/sdk/operations/add-asset.ts

```diff
@@ -108,10 +108,12 @@ export function addAsset(engine: IEngine) {
                 ...condition,
                 id: mapId(condition.id)
               })),
-              actions: trigger.actions.map((action) => ({
-                ...action,
-                id: mapId(action.id)
-              }))
+              actions: trigger.actions
+                .filter((action) => action.id)
```

Comment:
which asset contains actions without an id?

Reply:
trigger area smart item comes "empty"

<img width="322" alt="Screen Shot 2023-11-01 at 10 42 40" src="https://github.com/decentraland/js-sdk-toolchain/assets/2366069/6ba37ead-d8a3-4b61-b9f3-564ab480ea98">

Reply:
mm i think it would be better to remove those from the asset rather than filtering them here

Reply:
https://github.com/decentraland/asset-packs/blob/main/packs/smart_items/assets/trigger_area/data.json#L23-L26

Reply:
it would require a PR + release over there, and then upgrade the dependency over here

Reply:
removed empty actions from those components + added a validation rule for future cases:
https://github.com/decentraland/asset-packs/pull/48